.github/workflows/schema-check.yml

```diff
@@ -21,6 +21,8 @@ on:
       - "*.latest"
       - "releases/*"
 
+# no permissions beyond the default needed here
```

Comment:
Can we just put in the permission here?  I think it's a good standard to always specify the permissions, even when they match the defaults.

Reply:
That's a good point. Done!